network/dnscrypt-proxy/dnscrypt-proxy.SlackBuild

```diff
@@ -97,11 +97,13 @@ mkdir -p $PKG/usr/{bin,doc/$PRGNAM-$VERSION} \
     $PKG/var/{log,run}/$PRGNAM \
     $PKG/install
 
-install -D -m 0644 LICENSE $CWD/README.Slackware $CWD/$PRGNAM.SlackBuild \
-    $PKG/usr/doc/$PRGNAM-$VERSION/
-install -D -m 0644 $PRGNAM $PKG/usr/bin/ 
-install -D -m 0644 $CWD/$PRGNAM.toml $PKG/etc/$PRGNAM/
-install -D -m 0644 $CWD/rc.$PRGNAM $PKG/etc/rc.d/
+cp -a *.txt $PKG/etc/$PRGNAM/
```

Comment:
etc should be used only for configuration files
what is the content of those txt files?

Reply:
They are sample configuration files